client/src/modules/stock/inventories/registry.js

```diff
@@ -167,6 +168,15 @@ function StockInventoriesController(
     gridColumns.openConfigurationModal();
   }
 
+  function setDefaultFilters() {
+    const assignedKeys = Object.keys(stockInventoryFilters.formatHTTP());
+
+    // assign default includeEmptyLot filter
+    if (assignedKeys.indexOf('includeEmptyLot') === -1) {
```

Comment:
I always prefer [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) since it is more readable.

Reply:
FYI - the reason this failed before is because
```js
assignedKeys.indexOf('includeEmptyLot') === -1
```
is 
```js
!assignedKeys.includes('includeEmptyLot')
```

This is fine though :)